actions/notify-slack/get-info.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+load get-info.sh
+
+function setup() {
+    export GITHUB_OUTPUT="$BATS_TEST_TMPDIR/output"
+}
+
+function teardown() {
+    :
+}
+
+@test "it should not change EMOJI if set" {
+    EMOJI=test
+    run get-info
+
+    [ "$status" -eq 0 ]
+    grep -q 'emoji=test' "$GITHUB_OUTPUT"
+}
+
+@test "it should mot change MESSAGE if set." {
```

Comment:
```suggestion
@test "it should not change MESSAGE if set" {
```

Please don't add the period at the end of tests.  Even though it is allowed, we do not do that on any other test.